fs_s3fs/_s3fs.py

```diff
@@ -381,6 +392,8 @@ def s3(self):
                 aws_access_key_id=self.aws_access_key_id,
                 aws_secret_access_key=self.aws_secret_access_key,
                 aws_session_token=self.aws_session_token,
+                use_ssl=self.use_ssl,
+                verify=self.verify,
```

Comment:
The use_ssl and verify arguments need to be applied below as well, starting on line 410 (in the "client" method).

Reply:
oops! thanks for catching my silly oversight. fixed